### ISSUE TITLE
2025-04-10, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#136]
# Today's Arxiv Papers

(1) [Sculpting Subspaces: Constrained Full Fine-Tuning in LLMs for Continual Learning](http://arxiv.org/abs/2504.07097v1), 2025-04-09

(2) [OmniCaptioner: One Captioner to Rule Them All](http://arxiv.org/abs/2504.07089v1), 2025-04-09

(3) [KG-LLM-Bench: A Scalable Benchmark for Evaluating LLM Reasoning on Textualized Knowledge Graphs](http://arxiv.org/abs/2504.07087v1), 2025-04-09

(4) [DeduCE: Deductive Consistency as a Framework to Evaluate LLM Reasoning](http://arxiv.org/abs/2504.07080v1), 2025-04-09

(5) [A Survey on Personalized and Pluralistic Preference Alignment in Large Language Models](http://arxiv.org/abs/2504.07070v1), 2025-04-09

(6) [HalluciNot: Hallucination Detection Through Context and Common Knowledge Verification](http://arxiv.org/abs/2504.07069v1), 2025-04-09

(7) [TASTE: Text-Aligned Speech Tokenization and Embedding for Spoken Language Modeling](http://arxiv.org/abs/2504.07053v1), 2025-04-09

(8) [To Backtrack or Not to Backtrack: When Sequential Search Limits Model Reasoning](http://arxiv.org/abs/2504.07052v1), 2025-04-09

(9) [Distilling Textual Priors from LLM to Efficient Image Fusion](http://arxiv.org/abs/2504.07029v1), 2025-04-09

(10) [Evaluating Retrieval Augmented Generative Models for Document Queries in Transportation Safety](http://arxiv.org/abs/2504.07022v1), 2025-04-09

(11) [LLM-IFT: LLM-Powered Information Flow Tracking for Secure Hardware](http://arxiv.org/abs/2504.07015v1), 2025-04-09

(12) [Towards LLMs Robustness to Changes in Prompt Format Styles](http://arxiv.org/abs/2504.06969v1), 2025-04-09

(13) [VideoChat-R1: Enhancing Spatio-Temporal Perception via Reinforcement Fine-Tuning](http://arxiv.org/abs/2504.06958v1), 2025-04-09

(14) [RuOpinionNE-2024: Extraction of Opinion Tuples from Russian News Texts](http://arxiv.org/abs/2504.06947v1), 2025-04-09

(15) [Review of Case-Based Reasoning for LLM Agents: Theoretical Foundations, Architectural Components, and Cognitive Integration](http://arxiv.org/abs/2504.06943v1), 2025-04-09

(16) [FeedbackEval: A Benchmark for Evaluating Large Language Models in Feedback-Driven Code Repair Tasks](http://arxiv.org/abs/2504.06939v1), 2025-04-09

(17) [Data Augmentation for Fake Reviews Detection in Multiple Languages and Multiple Domains](http://arxiv.org/abs/2504.06917v1), 2025-04-09

(18) [Identifying Aspects in Peer Reviews](http://arxiv.org/abs/2504.06910v1), 2025-04-09

(19) [MovSAM: A Single-image Moving Object Segmentation Framework Based on Deep Thinking](http://arxiv.org/abs/2504.06863v1), 2025-04-09

(20) [EIDT-V: Exploiting Intersections in Diffusion Trajectories for Model-Agnostic, Zero-Shot, Training-Free Text-to-Video Generation](http://arxiv.org/abs/2504.06861v1), 2025-04-09

(21) [Integrating Cognitive Processing Signals into Language Models: A Review of Advances, Applications and Future Directions](http://arxiv.org/abs/2504.06843v1), 2025-04-09

(22) [LVC: A Lightweight Compression Framework for Enhancing VLMs in Long Video Understanding](http://arxiv.org/abs/2504.06835v1), 2025-04-09

(23) [Open Problems and a Hypothetical Path Forward in LLM Knowledge Paradigms](http://arxiv.org/abs/2504.06823v1), 2025-04-09

(24) [Zero-Shot Image-Based Large Language Model Approach to Road Pavement Monitoring](http://arxiv.org/abs/2504.06785v1), 2025-04-09

(25) [CHIME: A Compressive Framework for Holistic Interest Modeling](http://arxiv.org/abs/2504.06780v1), 2025-04-09

(26) [FamilyTool: A Multi-hop Personalized Tool Use Benchmark](http://arxiv.org/abs/2504.06766v1), 2025-04-09

(27) [Patch Matters: Training-free Fine-grained Image Caption Enhancement via Local Perception](http://arxiv.org/abs/2504.06666v1), 2025-04-09

(28) [SEE: Continual Fine-tuning with Sequential Ensemble of Experts](http://arxiv.org/abs/2504.06664v1), 2025-04-09

(29) [Bridging the Gap Between Preference Alignment and Machine Unlearning](http://arxiv.org/abs/2504.06659v1), 2025-04-09

(30) [A Neuro-inspired Interpretation of Unlearning in Large Language Models through Sample-level Unlearning Difficulty](http://arxiv.org/abs/2504.06658v1), 2025-04-09

(31) [ThoughtProbe: Classifier-Guided Thought Space Exploration Leveraging LLM Intrinsic Reasoning](http://arxiv.org/abs/2504.06650v1), 2025-04-09

(32) [SCI-Reason: A Dataset with Chain-of-Thought Rationales for Complex Multimodal Reasoning in Academic Areas](http://arxiv.org/abs/2504.06637v1), 2025-04-09

(33) [BBQRec: Behavior-Bind Quantization for Multi-Modal Sequential Recommendation](http://arxiv.org/abs/2504.06636v1), 2025-04-09

(34) [The Method for Storing Patterns in Neural Networks-Memorization and Recall of QR code Patterns-](http://arxiv.org/abs/2504.06631v1), 2025-04-09

(35) [AgentFM: Role-Aware Failure Management for Distributed Databases with LLM-Driven Multi-Agents](http://dx.doi.org/10.1145/3696630.3728492), 2025-04-09

(36) [Benchmarking Multimodal CoT Reward Model Stepwise by Visual Program](http://arxiv.org/abs/2504.06606v1), 2025-04-09

(37) [Automated Business Process Analysis: An LLM-Based Approach to Value Assessment](http://arxiv.org/abs/2504.06600v1), 2025-04-09

(38) [A Multi-Modal Interaction Framework for Efficient Human-Robot Collaborative Shelf Picking](http://arxiv.org/abs/2504.06593v1), 2025-04-09

(39) [Right Prediction, Wrong Reasoning: Uncovering LLM Misalignment in RA Disease Diagnosis](http://arxiv.org/abs/2504.06581v1), 2025-04-09

(40) [Bypassing Safety Guardrails in LLMs Using Humor](http://arxiv.org/abs/2504.06577v1), 2025-04-09

(41) [Defending LLM Watermarking Against Spoofing Attacks with Contrastive Representation Learning](http://arxiv.org/abs/2504.06575v1), 2025-04-09

(42) [Do Reasoning Models Show Better Verbalized Calibration?](http://arxiv.org/abs/2504.06564v1), 2025-04-09

(43) [FuseRL: Dense Preference Optimization for Heterogeneous Model Fusion](http://arxiv.org/abs/2504.06562v1), 2025-04-09

(44) [NeedleInATable: Exploring Long-Context Capability of Large Language Models towards Long-Structured Tables](http://arxiv.org/abs/2504.06560v1), 2025-04-09

(45) [ASHiTA: Automatic Scene-grounded HIerarchical Task Analysis](http://arxiv.org/abs/2504.06553v1), 2025-04-09

(46) [Lugha-Llama: Adapting Large Language Models for African Languages](http://arxiv.org/abs/2504.06536v1), 2025-04-09

(47) [Missing Premise exacerbates Overthinking: Are Reasoning Models losing Critical Thinking Skill?](http://arxiv.org/abs/2504.06514v1), 2025-04-09

(48) [GTS-LUM: Reshaping User Behavior Modeling with LLMs in Telecommunications Industry](http://arxiv.org/abs/2504.06511v1), 2025-04-09

(49) [Mind the Gap: Evaluating Vision Systems in Small Data Applications](http://arxiv.org/abs/2504.06486v1), 2025-04-08

(50) [Can LLMs Simulate Personas with Reversed Performance? A Benchmark for Counterfactual Instruction Following](http://arxiv.org/abs/2504.06460v1), 2025-04-08

(51) [Can you Finetune your Binoculars? Embedding Text Watermarks into the Weights of Large Language Models](http://arxiv.org/abs/2504.06446v1), 2025-04-08

(52) [Don't Let It Hallucinate: Premise Verification via Retrieval-Augmented Logical Reasoning](http://arxiv.org/abs/2504.06438v1), 2025-04-08

(53) [Language-Dependent Political Bias in AI: A Study of ChatGPT and Gemini](http://arxiv.org/abs/2504.06436v1), 2025-04-08

(54) [Human Trust in AI Search: A Large-Scale Experiment](http://arxiv.org/abs/2504.06435v1), 2025-04-08

(55) [S'MoRE: Structural Mixture of Residual Experts for LLM Fine-tuning](http://arxiv.org/abs/2504.06426v1), 2025-04-08

(56) [SPIRe: Boosting LLM Inference Throughput with Speculative Decoding](http://arxiv.org/abs/2504.06419v1), 2025-04-08

(57) [The Zero Body Problem: Probing LLM Use of Sensory Language](http://arxiv.org/abs/2504.06393v1), 2025-04-08

(58) [Comparing Self-Disclosure Themes and Semantics to a Human, a Robot, and a Disembodied Agent](http://arxiv.org/abs/2504.06374v1), 2025-04-08

(59) [Query Understanding in LLM-based Conversational Information Seeking](http://arxiv.org/abs/2504.06356v1), 2025-04-08

(60) [GOLLuM: Gaussian Process Optimized LLMs -- Reframing LLM Finetuning through Bayesian Optimization](http://arxiv.org/abs/2504.06265v1), 2025-04-08

(61) [Hogwild! Inference: Parallel LLM Generation via Concurrent Attention](http://arxiv.org/abs/2504.06261v2), 2025-04-08

(62) [FEABench: Evaluating Language Models on Multiphysics Reasoning Ability](http://arxiv.org/abs/2504.06260v1), 2025-04-08

(63) [Transfer between Modalities with MetaQueries](http://arxiv.org/abs/2504.06256v1), 2025-04-08

(64) [LExT: Towards Evaluating Trustworthiness of Natural Language Explanations](http://arxiv.org/abs/2504.06227v1), 2025-04-08

(65) [Encoder-Decoder Gemma: Improving the Quality-Efficiency Trade-Off via Adaptation](http://arxiv.org/abs/2504.06225v1), 2025-04-08

(66) [Can Performant LLMs Be Ethical? Quantifying the Impact of Web Crawling Opt-Outs](http://arxiv.org/abs/2504.06219v1), 2025-04-08

(67) [From 128K to 4M: Efficient Training of Ultra-Long Context Large Language Models](http://arxiv.org/abs/2504.06214v1), 2025-04-08

(68) [TxGemma: Efficient and Agentic LLMs for Therapeutics](http://arxiv.org/abs/2504.06196v1), 2025-04-08

(69) [Assessing how hyperparameters impact Large Language Models' sarcasm detection performance](http://arxiv.org/abs/2504.06166v1), 2025-04-08

(70) [Navigating the Rabbit Hole: Emergent Biases in LLM-Generated Attack Narratives Targeting Mental Health Groups](http://arxiv.org/abs/2504.06160v2), 2025-04-08

(71) [V-MAGE: A Game Evaluation Framework for Assessing Visual-Centric Capabilities in Multimodal Large Language Models](http://arxiv.org/abs/2504.06148v1), 2025-04-08

(72) [ARLO: A Tailorable Approach for Transforming Natural Language Software Requirements into Architecture using LLMs](http://arxiv.org/abs/2504.06143v1), 2025-04-08

(73) [QGen Studio: An Adaptive Question-Answer Generation, Training and Evaluation Platform](http://arxiv.org/abs/2504.06136v1), 2025-04-08

(74) [Knowledge Graph Completion with Relation-Aware Anchor Enhancement](http://arxiv.org/abs/2504.06129v1), 2025-04-08

(75) [Leanabell-Prover: Posttraining Scaling in Formal Reasoning](http://arxiv.org/abs/2504.06122v2), 2025-04-08

(76) [Nonuniform-Tensor-Parallelism: Mitigating GPU failure impact for Scaled-up LLM Training](http://arxiv.org/abs/2504.06095v1), 2025-04-08

(77) [Multi-Sense Embeddings for Language Models and Knowledge Distillation](http://arxiv.org/abs/2504.06036v1), 2025-04-08

(78) [A Geometric-Aware Perspective and Beyond: Hybrid Quantum-Classical Machine Learning Methods](http://arxiv.org/abs/2504.06328v1), 2025-04-08

(79) [CAI: An Open, Bug Bounty-Ready Cybersecurity AI](http://arxiv.org/abs/2504.06017v2), 2025-04-08

(80) [Llama-3-Nanda-10B-Chat: An Open Generative Large Language Model for Hindi](http://arxiv.org/abs/2504.06011v1), 2025-04-08

(81) [Optuna vs Code Llama: Are LLMs a New Paradigm for Hyperparameter Tuning?](http://arxiv.org/abs/2504.06006v1), 2025-04-08

(82) [NativQA Framework: Enabling LLMs with Native, Local, and Everyday Knowledge](http://arxiv.org/abs/2504.05995v1), 2025-04-08

(83) [Unsupervised Location Mapping for Narrative Corpora](http://arxiv.org/abs/2504.05954v1), 2025-04-08

(84) [InstructMPC: A Human-LLM-in-the-Loop Framework for Context-Aware Control](http://arxiv.org/abs/2504.05946v1), 2025-04-08

(85) [From Stability to Inconsistency: A Study of Moral Preferences in LLMs](http://arxiv.org/abs/2504.06324v1), 2025-04-08

(86) [Mosaic: Composite Projection Pruning for Resource-efficient LLMs](http://arxiv.org/abs/2504.06323v1), 2025-04-08

(87) [Assessing Thai Dialect Performance in LLMs with Automatic Benchmarks and Human Evaluation](http://arxiv.org/abs/2504.05898v1), 2025-04-08

(88) [HybriMoE: Hybrid CPU-GPU Scheduling and Cache Management for Efficient MoE Inference](http://arxiv.org/abs/2504.05897v1), 2025-04-08

(89) [Agent Guide: A Simple Agent Behavioral Watermarking Framework](http://arxiv.org/abs/2504.05871v1), 2025-04-08

(90) [CTI-HAL: A Human-Annotated Dataset for Cyber Threat Intelligence Analysis](http://arxiv.org/abs/2504.05866v1), 2025-04-08

(91) [Are Generative AI Agents Effective Personalized Financial Advisors?](http://arxiv.org/abs/2504.05862v1), 2025-04-08

(92) [Enhancing Coreference Resolution with Pretrained Language Models: Bridging the Gap Between Syntax and Semantics](http://arxiv.org/abs/2504.05855v1), 2025-04-08

(93) [PathGPT: Leveraging Large Language Models for Personalized Route Generation](http://arxiv.org/abs/2504.05846v1), 2025-04-08

(94) [Accelerating LLM Inference Throughput via Asynchronous KV Cache Prefetching](http://arxiv.org/abs/2504.06319v1), 2025-04-08

(95) [Leveraging Robust Optimization for LLM Alignment under Distribution Shifts](http://arxiv.org/abs/2504.05831v1), 2025-04-08

(96) [Right Question is Already Half the Answer: Fully Unsupervised LLM Reasoning Incentivization](http://arxiv.org/abs/2504.05812v1), 2025-04-08

(97) [PaMi-VDPO: Mitigating Video Hallucinations by Prompt-Aware Multi-Instance Video Preference Learning](http://arxiv.org/abs/2504.05810v1), 2025-04-08

(98) [StealthRank: LLM Ranking Manipulation via Stealthy Prompt Optimization](http://arxiv.org/abs/2504.05804v1), 2025-04-08

(99) [From Superficial to Deep: Integrating External Knowledge for Follow-up Question Generation Using Knowledge Graph and LLM](http://arxiv.org/abs/2504.05801v1), 2025-04-08

(100) [How to Enable LLM with 3D Capacity? A Survey of Spatial Reasoning in LLM](http://arxiv.org/abs/2504.05786v1), 2025-04-08
